### ISSUE TITLE
how to calculate the scale, can you provide me the script to write csv?

### ISSUE THREAD

**Issue by wqz960** [HRNet/HRNet-Facial-Landmark-Detection#10]
Thank you for your code!
Could you please provide the script to write csv? Thank you!

**Comment by wqz960** [comment]
another question: When I use two gpus, how to change the learning rate? lr*2?

**Comment by yangyangkiki** [comment]
please find the answer in [https://github.com/HRNet/HRNet-Facial-Landmark-Detection/issues/3](url)

**Comment by wqz960** [comment]
@yangyangkiki Is there some tricks I forgot to add? I followed your Readme, but the training on 300W dataset was not the same with the log you provided. Finally, the nme for 300W validation is about 0.04, which has big difference between your paper's, and as for config file, I have changed the validation set to 300W_validation, in you paper , the nme is 0.0322. Please help me ! Really thanks!

**Comment by yangyangkiki** [comment]
Our code has been updated.
